base/uk.ac.stfc.isis.ibex.ui.configserver/src/uk/ac/stfc/isis/ibex/ui/configserver/dialogs/MultipleConfigsSelectionDialog.java

```diff
@@ -105,30 +106,39 @@ public MultipleConfigsSelectionDialog(
 		return selected;
 	}
 
-	@Override
-	protected void okPressed() {
-        selected = asString(items.getSelection());
-		super.okPressed();
-	}
+    @Override
+   protected void okPressed() {
+       Collection<String> selectedItems = asString(items.getSelection());
+       
+       /* The following ensures that double clicking on the description/white space doesn't
+        * launch the process using a description/null as the name.
+        */    
+       List<String> names = (List<String>) ConfigInfo.names(available);
+       boolean anyMatch = selectedItems.stream().anyMatch(names::contains);
+       if (anyMatch) {
+    	   selected = asString(items.getSelection());
+    	   super.okPressed();
+       }
+   }
 
 	@Override
     protected void createSelection(Composite container) {
 	    
 		Label lblSelect = new Label(container, SWT.NONE);
         lblSelect.setText("Select " + getTypeString() + ":");
-        items = createTable(container, SWT.BORDER | SWT.V_SCROLL | extraListOptions);
-        
+        List<String> columnNames = Arrays.asList("Name", "Description");
+        items = createTable(container, SWT.BORDER | SWT.V_SCROLL | extraListOptions, columnNames);
 
-        String[] names;
-        String[] descriptions;
+        List<String> names;
+        List<String> descriptions;
         if (includeCurrent) {
-            names = ConfigInfo.names(available).toArray(new String[0]);
-            descriptions = ConfigInfo.descriptions(available).toArray(new String[0]);
+            names = (List<String>) ConfigInfo.names(available);
+            descriptions = (List<String>) ConfigInfo.descriptions(available);
         } else {
-            names = ConfigInfo.namesWithoutCurrent(available).toArray(new String[0]);
-            descriptions = ConfigInfo.descriptionsWithoutCurrent(available).toArray(new String[0]);
+            names = (List<String>) ConfigInfo.namesWithoutCurrent(available);
+            descriptions = (List<String>) ConfigInfo.descriptionsWithoutCurrent(available);
```

Comment:
There is something wrong in an API here, we shouldn't need this cast.

Either make `ConfigInfo.names(...)` and `ConfigInfo.descriptions(...)` return `List` rather than collection if they are already ordered, or we need to do something like `names = new ArrayList(ConfigInfo.names(...))`. The cast is brittle and may result in hard-to-trace bugs later if the API gets updated.

Feel free to grab me if you need to chat.